bin/con_man_gui.rb

```diff
@@ -2,10 +2,12 @@
 
 $LOAD_PATH.unshift File.dirname(__FILE__) + '/../lib'
 
-require 'con_man'
+require 'add_contact'
+require 'sql_contacts_repository'
 require 'qt'
 
+contacts_repository = SQLContactsRepository.new
 app = Qt::Application.new (ARGV)
-window = ConMan.new
+window = AddContact.new(contacts_repository)
```

Comment:
For widget classes and the like it's usually best to suffix the type name onto the end, e.g. `AddContactWindow`. Class names should (usually) be a noun i.e. representing a 'thing' in your object model.